src/nodes/primitive/range.ts

```diff
@@ -28,32 +33,32 @@ export const comparators = {
 export type Comparator = keyof typeof comparators
 
 export const numericComparatorDescriptions = {
-    "<": "less than",
-    ">": "more than",
-    "<=": "at most",
-    ">=": "at least",
-    "==": "exactly"
+    "<": "less than ",
```

Comment:
If you think about this from an abstract perspective, you wouldn't expect comparator descriptions to have a space at the end. Though it is useful in some contexts, if you didn't want it you'd have to then remember to trim the string, and it would be easy to assume it's not there and add a second space.

Best to handle this at the locations the description is referenced, which is generally fairly convenient from a template literal anyways.